solidity/test/token_stake/TestTokenStakeLock.js

```diff
@@ -153,13 +137,12 @@ describe('TokenStaking/Lock', () => {
 
     it("should not permit locks that exceed the maximum lock duration", async () => {
       await time.increaseTo(initializationPeriod.add(createdAt).addn(1))
-      let maximumDuration = await stakingContract.maximumLockDuration();
-      let longPeriod = maximumDuration.addn(1);
+      const maximumDuration = time.duration.days(200);
```

Comment:
Why no longer read the value from the contract?

Reply:
This value is no longer public on the operator contract. It exists in the library and only the library uses it. To make it public, I need to expose a public function on the operator contract that will cost some bytecode. Given that it's just one unit test, I don't think it's crazy to duplicate it. At least, we are also testing it's not some crazy 1-minute value. Every byte matters given the code we want to add to `TokenStaking`.